src/pages/index.js

```diff
@@ -38,26 +41,79 @@ const modalImageCloseButton = previewImageModal.querySelector(
   "#modal__image-close-button"
 );
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "65e7222e-9077-40ab-ad47-86d360e310e2",
+    "Content-Type": "application/json",
+  },
+});
+
+const editAvatarPopup = new PopupWithForm({
+  popupSelector: "#edit-avatar-modal",
+  handleFormSubmit: handleAvatarFormSubmit,
+});
+editAvatarPopup.setEventListeners();
+
+const profileAvatarForm = document.forms["avatar-form"];
+const profileAvatarButton = document.querySelector(".profile__avatar-button");
+const avatarFormValidator = new FormValidator(config, profileAvatarForm);
+avatarFormValidator.enableValidation();
+
+let userId;
+
+profileAvatarButton.addEventListener("click", () => {
+  editAvatarPopup.open();
+  avatarFormValidator._resetValidation();
```

Comment:
`_resetValidation`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.